<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>399</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>399</width>
    <height>516</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>399</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Toolchain Manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="projectDirLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Default projects directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="projectDirEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="projectDirButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="fontLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Editor font:</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="currentFont">
      <font>
       <family>Nimbus Mono PS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="fontSizeBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>53</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>72</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="darkThemeCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable dark theme-friendly text editor color scheme</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="toolchainsTab">
    <attribute name="title">
     <string>Toolchains</string>
    </attribute>
    <widget class="QLabel" name="toolchainsHeaderLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Registered toolchains:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addToolchainButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset resource="MuleStudioResources.qrc">
       <normaloff>:/oxygenicons/resources/icons/list-add.png</normaloff>:/oxygenicons/resources/icons/list-add.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="editToolchainButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit config</string>
     </property>
     <property name="icon">
      <iconset resource="MuleStudioResources.qrc">
       <normaloff>:/oxygenicons/resources/icons/document-edit.png</normaloff>:/oxygenicons/resources/icons/document-edit.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="removeToolchainButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset resource="MuleStudioResources.qrc">
       <normaloff>:/oxygenicons/resources/icons/list-remove.png</normaloff>:/oxygenicons/resources/icons/list-remove.png</iconset>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>271</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aboutTab">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="MuleStudioResources.qrc">:/mulestudiointernalicons/resources/icons/mulestudiologo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="muleStudioAboutLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;libMule Toolchain Manager release ^V^&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Copyright (c) Tim K &amp;lt;&lt;a href=&quot;mailto:timprogrammer@rambler.ru&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;timprogrammer@rambler.ru&lt;/span&gt;&lt;/a&gt;&amp;gt; 2018. &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Licensed under GNU LGPLv2.1.&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Oxygen icons were drawn by awesome designers from KDE &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;project: &lt;a href=&quot;http://kde.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://kde.org/.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MuleStudioResources.qrc"/>
 </resources>
 <connections/>
</ui>
